### PR TITLE
Edit image like community with popup

### PR DESCRIPTION
**Pull request by mghazanfar**
This PR includes:
- A popup to edit the image
- Changing image from popup changes the image everywhere without reload
- Changing image from form changes the image everywhere without reload

### CODE REVIEW

File: src/common/components/profile-card/_index.scss [hunk 1]
```diff
@@ -227,3 +227,27 @@
...
+  cursor: pointer;
+
+  &:hover {
+    box-shadow: $box-shadow;
+  }
+
+  svg {
+    height: 16px;
+  }
+}
```
Comment (feruzm) [inline src/common/components/profile-card/_index.scss:253]: end file with new line

File: src/common/components/profile-card/index.tsx [hunk 2]
```diff
@@ -36,11 +36,13 @@ import {
...
 } from "../../img/svg";
 
+import { EditPic } from '../community-card'
+
 interface Props {
     global: Global;
     history: History;
     activeUser: ActiveUser | null;
-    account: Account;
+    account: any;
```
Comment (feruzm) [inline src/common/components/profile-card/index.tsx:45]: Why type `any`?

File: src/common/components/profile-edit/index.tsx [hunk 3]
```diff
@@ -16,6 +16,7 @@ import {getAccount} from "../../api/hive";
 
 interface Props {
     activeUser: ActiveUser;
+    account: any;
```
Comment (feruzm) [inline src/common/components/profile-edit/index.tsx:19]: Also why `any`?

File: src/common/components/profile-edit/index.tsx [hunk 4]
```diff
@@ -87,12 +96,12 @@ export default class ProfileEdit extends BaseComponent<Props, State> {
         }).then((account) => {
 
             // reload page to refresh profile image
-            if (activeUser.data.__loaded && activeUser.data.profile?.profile_image !== account.profile?.profile_image) {
-                setTimeout(() => {
-                    window.location.reload();
-                }, 500);
-                return;
-            }
+            // if (activeUser.data.__loaded && activeUser.data.profile?.profile_image !== account.profile?.profile_image) {
```
Comment (feruzm) [inline src/common/components/profile-edit/index.tsx:99]: reload not needed anymore right? we can remove this part instead of comment if that's the case

**Comment by feruzm** [comment]
@mghazanfar also after update, I am getting error message. Top navigation User menu part not updating avatar, only after reload of page, maybe trigger re-render on navbar or user menu component for it to fetch new image?
 
image

File: src/common/components/profile-card/index.tsx (hunk 2 above)
Comment (mghazanfar) [reply src/common/components/profile-card/index.tsx:45]: Neither Account nor FullAccount | BaseAccount were working for getting profile info. I work on it more.

File: src/common/components/profile-edit/index.tsx (hunk 3 above)
Comment (mghazanfar) [reply src/common/components/profile-edit/index.tsx:19]: Same issue as above.

**Comment by mghazanfar** [comment]
> @mghazanfar also after update, I am getting error message. Top navigation User menu part not updating avatar, only after reload of page, maybe trigger re-render on navbar or user menu component for it to fetch new image?
> 
> image

On settings page, it changes as both the images. But when we upload image on profile page, data does not update immediately in the avatar inside navbar but it makes a call to getAccounts which changes picture in navbar after a few moments.

**Review by feruzm — APPROVED** [review]

**Pull request merged**